oremda/cli/runner.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+import click
+from oremda.pipeline.runner import start
+
+
+@click.command(
+    "runner",
+    short_help="oremda runner",
```

Comment:
We might want a better name here!

Reply:
So currently  `oremda run` can be using to run a pipeline from a JSON file. The "runner" is a separate service that communicates with web server to run pipelines executed from the web UI.